Resources/Prototypes/_Moffstation/Roles/Antags/vampire.yml

```
@@ -0,0 +1,9 @@
+﻿- type: antag
+  id: Vampire
+  name: roles-antag-vampire-name
+  antagonist: true
+  setPreference: true
+  objective: roles-antag-vampire-objective
+  requirements:
+  - !type:OverallPlaytimeRequirement
+    time: 18000 # 5h
```

I looked into this recently, and I think you can actually put literally
```suggestion
    time: 5h
```
because the deserializer is smart